daos/private/testdao.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Sagacious Senor",
+  "name": "Obtuse Artisan",
```

Comment:
does this file need to be updated or it can be added to .gitignore ?

Reply:
Yes it needs as we use it in the tests.